app/src/docs/assets/Icons.doc.tsx

```diff
@@ -125,24 +130,26 @@ export class IconsDoc extends React.Component {
                         <IconButton onClick={ () => {} } icon={ icon }/>
                     </FlexCell>
                     <FlexCell width='auto' shrink={ 0 } >
-                        <Button size={ this.state.controlSize } onClick={ () => {} } icon={ icon }/>
+                        <Button size={ this.state.controlSize } onClick={ () => {} } icon={ icon } />
                     </FlexCell>
                     <FlexCell width='auto' shrink={ 0 } >
-                        <Button caption='Click' size={ this.state.controlSize } onClick={ () => {} } icon={ icon }/>
+                        <Button caption='Click' size={ this.state.controlSize } onClick={ () => {} } icon={ icon } />
                     </FlexCell>
                     <FlexCell width='auto' shrink={ 0 } >
-                        <LinkButton caption='Click' size={ this.state.controlSize } onClick={ () => {} } icon={ icon }/>
+                        <LinkButton caption='Click' size={ this.state.controlSize } onClick={ () => {} } icon={ icon } />
                     </FlexCell>
                 </FlexRow>
-                <FlexRow size='24'  vPadding='24'>
-                    <TextInput value='Some text' size={ this.state.controlSize } onValueChange={ () => {} } icon={ icon }/>
+                <FlexRow size='24' vPadding='24'>
+                    <TextInput value='Some text' size={ this.state.controlSize } onValueChange={ () => {} } icon={ icon } />
                 </FlexRow>
             </FlexCell>
         );
     }
 
     checkValidSize = () => {
-        if (typeof recommendedSizes[this.state.selectedIcon.size] === 'undefined') {
+        if (this.state.selectedIcon.name) {
```

Comment:
why do we have here condition with `selectedIcon.name`? We need check about if we have icon with only invalid sizes, we don't need to show this warning.